bluej/src/main/java/bluej/utility/javafx/DelegableScalableTextField.java

```diff
@@ -63,18 +72,40 @@ public class DelegableScalableTextField<DELEGATE_IDENT> extends ScalableHeightTe
 
     private final TextFieldDelegate<DELEGATE_IDENT> delegate;
     private final DELEGATE_IDENT delegateId;
+
+    // Keep track of InputMethod (IM) positions, e.g. for entering Chinese with
+    // the on-screen IME keyboard.  imStart is the start of the current sequence of
+    // "composing" characters that could end up staying as English or transformed into
```

Comment:
If I understood it right, shall we specify "imStart is the start [..] _in the current text slot_."